.env.example

```
@@ -0,0 +1,158 @@
+# CORS origin - Base URL for API requests
+# Required by: All apps
+NEXT_PUBLIC_CORS_ORIGIN="http://localhost:3002"
+
+# Allowed origin pattern for CORS validation
+# Can be "*", a URL, or wildcard pattern like "*.domain.com"
+# Required by: All apps
+NEXT_PUBLIC_ALLOWED_ORIGIN="*"
+
+# Telegram bot name
+# Required by: All apps (base)
+TELEGRAM_BOT_NAME="your_bot_name"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)